api/src/lib/aws.ts

```diff
@@ -64,7 +64,7 @@ export function uploadWorkbook(
   uploadId: number,
   body: StreamingBlobPayloadInputTypes
 ) {
-  const folderName = `uploads/${upload.organizationId}/${upload.agencyId}/${upload.reportingPeriodId}/${upload.expenditureCategoryId}/${uploadId}/${upload.filename}`
+  const folderName = `uploads/${upload.organizationId}/${upload.agencyId}/${upload.reportingPeriodId}/${uploadId}/${upload.filename}`
```

Comment:
Related question for #135 - how stable do you expect these paths will be? Should I always find uploadId as the last node before filename?

Reply:
We expect this to be set in stone. I'm expecting the removal of the expenditure category code to be the last change needed here.